metrics/timer.go

```diff
@@ -6,24 +6,26 @@ import (
 
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
 )
 
 // NewTimerMs creates a Float64Timer with units of milliseconds and a default set of aggregation
 // bounds for latencies up to a few seconds.
-func NewTimerMs(name, desc string) *Float64Timer {
+func NewTimerMs(name, desc string, tagKeys ...tag.Key) *Float64Timer {
```

Comment:
we need to be able to associate a tag with an opencensus metric, this is how we plumb that through.